src/model_signing/verifying.py

```diff
@@ -35,31 +37,14 @@
     from typing_extensions import Self
 
 
-def verify(
-    model_path: os.PathLike,
-    signature_path: os.PathLike,
-    *,
-    identity: str,
-    oidc_issuer: Optional[str] = None,
-    use_staging: bool = False,
-):
-    """Verifies that a model conforms to a signature.
-
-    Currently, this assumes signatures over DSSE, using Sigstore. We will add
-    support for more cases in a future change.
-
-    Args:
-        model_path: the path to the model to verify.
-        signature_path: the path to the signature to check.
-        identity: The expected identity that has signed the model.
-        oidc_issuer: The expected OpenID Connect issuer that provided the
-          certificate used for the signature.
-        use_staging: Use staging configurations, instead of production. This
-          is supposed to be set to True only when testing. Default is False.
+def verify(model_path: os.PathLike, signature_path: os.PathLike):
+    """Verifies that a model matches a signature in the default configuration.
+
+    There is no default configuration with default arguments that can be used,
+    so this just raises an exception to signal that verification configuration
+    needs to be set up.
```

Comment:
why have the function if it will always raise an exception?

Reply:
It will still raise an exception if you use the default config without setting a verifier. We don't have enough optional arguments here to set a default verifier